com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1474,7 +1474,7 @@ public void SetClientSecrets(string serverCommonName, string caCertificate = nul
                             }
                             else
                             {
-                                m_NetworkSettings.WithSecureClientParameters(m_ClientCaCertificate, m_ServerCommonName));
+                                m_NetworkSettings.WithSecureClientParameters(m_ClientCaCertificate, m_ServerCommonName);
```

Comment:
Ugh. How did that slip by ? :-)

Reply:
Probably a typo on my end when I updated to the new string-based APIs. I hadn't realized that encryption support was behind the UTP 2.0 define, so hadn't thought of trying to compile with that installed.